xarray/tests/test_formatting.py

```diff
@@ -101,6 +101,8 @@ def test_format_item(self) -> None:
             (np.float16(1.1234), "1.123"),
             (np.float32(1.0111111), "1.011"),
             (np.float64(22.222222), "22.22"),
+            (np.zeros((1, 1)), "0.0"),
```

Comment:
This is the original behavior.

I would argue that this is technically incorrect.

As an alternative we could check if `x.shape == ()`.

Reply:
I agree, let's do the shape check

Reply:
done! @dcherian, could you have another look?